src/pages/organizers/create/OrganizerForm.tsx

```diff
@@ -203,6 +203,7 @@ const OrganizerForm = () => {
           form={form}
           labels={organizerLabels}
           onChange={() => {
+            if (!form.getFieldState('nameAndUrl').isDirty) return;
```

Comment:
I recommend avoiding single-line if statements and always using curly braces. One-liners make it easy to overlook critical control flow like return and are prone to errors if someone later adds more logic, such as a logging statement, inside the if block.